<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ecu</class>
 <widget class="QMainWindow" name="ecu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ecu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QLineEdit" name="txtRPM">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>RPM</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTPS">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TPS</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Injeksi</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>FlowRate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFlow">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtInjVol">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Durasi t</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDurasiInjT">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Durasi deg</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDurasiInjD">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Off deg</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOffInjD">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>On deg</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOnInjD">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>On t</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOnInjT">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Pengapian</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Off deg</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOffIgnDeg">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOnIgnDeg">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>On deg</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDurasiIgnDeg">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Dur deg</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Dur t</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDurasiIgnT">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOnIgnT">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>On t</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCalc"/>
   <addaction name="actionQuit"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionCalc">
   <property name="text">
    <string>Calc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
